csprng.go

```diff
@@ -136,10 +53,10 @@ func NewDRNGwithSeed(password, realm string, seed []byte) (io.Reader, error) {
 		return nil, err
 	}
 
-	rng := &fortunaGenerator{}
-	rng.Reseed(rngSeed)
+	block, _ := aes.NewCipher(passKey(password, realm))
```

Comment:
Sorry, missed this last time.
I guess here should be `block, _ := aes.NewCipher(rngSeed)`, otherwise we are not using provided master seed.